src/Insect/Format.purs

```diff
@@ -89,7 +88,7 @@ uncurry fmt (Formatted ot ft s) = fmt ot ft s
 
 -- | Format an output text with a given formatter.
 format ∷ Formatter → Markup → String
-format formatter m = foldMap (uncurry formatter) (optional nl : m)
+format formatter m = foldMap (uncurry formatter) ([ optional nl ] <> m)
```

Comment:
why is this better?

Reply:
I'm not sure I can explain well, but whenever I see a `:` that's not for lists, it kind of trips me up. Additionally, it rids us of an import, though admittedly that's not a big deal.

Reply:
Ok, I see. You mean: cons seems to imply an O(1) operation, which is not the case for an array.

Reply:
Maybe that's it, but I'll be the first to say out loud that it's more of a gut reaction than anything based on facts, hehe.

Reply:
I'm fine with the new version too, no worries